packages/opentelemetry-metrics/src/MeterProvider.ts

```diff
@@ -54,4 +57,19 @@ export class MeterProvider implements api.MeterProvider {
 
     return this._meters.get(key)!;
   }
+
+  shutdown(cb: () => void = () => {}) {
+    this.shutdownAllMeters().then(() => {
+      setTimeout(cb, 0);
+    });
+  }
+
+  private async shutdownAllMeters() {
```

Comment:
style: private methods should have leading `_`
```suggestion
  private async _shutdownAllMeters() {
```